TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/http/retries/DefaultRetry.java

```diff
@@ -0,0 +1,54 @@
+package org.opensearch.migrations.replay.http.retries;
+
+import java.util.List;
+
+import org.opensearch.migrations.replay.AggregatedRawResponse;
+import org.opensearch.migrations.replay.IRequestResponsePacketPair;
+import org.opensearch.migrations.replay.RequestSenderOrchestrator;
+import org.opensearch.migrations.replay.util.TextTrackedFuture;
+import org.opensearch.migrations.replay.util.TrackedFuture;
+
+import io.netty.buffer.ByteBuf;
+
+public class DefaultRetry implements RequestRetryEvaluator {
+    static final int MAX_RETRIES = 4;
```

Comment:
Did we talk about this being infinite by default?

Reply:
You & I talked about infinite as the default for _connection_ errors and it still is.  This is for errors like 404 or 5xx that are retryable.  If there's a single request that causes issues, we'll want to give it some fair tries, but then move on.

Reply:
Sounds good